src/main/resources/META-INF/plugin.xml

```diff
@@ -5,6 +5,11 @@
 
     <depends>com.intellij.modules.lang</depends>
     <depends>com.intellij.modules.platform</depends>
+    <!--
+      Note that since 2022.3:
+      * the plugin *folder* under $PRODUCT_ROOT$/plugins/ is called `vcs-git` and not `git4idea`, but
+      * the <id> of the plugin, as recorded in plugin's META-INF/plugin.xml is still `Git4Idea` - and that's what's expected here in `<depends>`
```

Comment:
Unlike with `plugins.set(...)`, here `vcs-git` (i.e. folder name) isn't accepted

Reply:
cool. thanks for this comment